### DISCUSSION TITLE
How do I confirm a javascript prompt in Cypress?

### DISCUSSION THREAD

**Discussion by DiogoManzoli** [cypress-io/cypress#29218] [Questions and Help]
![image](https://github.com/cypress-io/cypress/assets/85167436/76701bf7-e292-4b61-bea7-222ee0d74997)

After I click on a button, this confirm window appears, but I didn't find anything that confirms this prompt

Part of the code that opens the prompt:
``` 
 cy.get("iframe")
      .its("0.contentDocument")
      .its("body")
      .find("#f_bln_assistencia_confirmar")
      .click();

```

Help me??

**Comment by samelawrence** [comment]
I believe this may be related to the Chrome team deprecating the `unload` event. It seems that Cypress no longer has the ability to automatically accept/dismiss `window:alert` objects. This bug is causing major issues for my team, and we would love a resolution. cc @jennifer-shehane

**Reply by samtsai** [reply]
A workaround may be to stub the event:
cy.on("window:confirm", cy.stub().returns(true))

If Cypress is no longer auto closing then yea I guess that's a bug, which as you mentioned @samelawrence  might be due to browser APIs changing. Does the confirm go away in Firefox?

**Reply by samelawrence** [reply]
@samtsai This issue has been reproduced by Cypress staff in all browsers and is tracked in [this issue](https://github.com/cypress-io/cypress/issues/27896).
